src/calculateRentalCost.js

```diff
@@ -4,7 +4,19 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  if (days >= 3 && days < 6) {
+    return 100;
```

Comment:
The function returns a hardcoded value of 100 when the number of days is between 3 and 5 inclusive. This does not follow the logic described in the task. The cost should be calculated as 40 * number of days - 20.